sections/video.liquid

```diff
@@ -0,0 +1,141 @@
+      >
+        {%- if section.settings.cover_image != blank -%}
+          <img
+            srcset="{%- if section.settings.cover_image.width >= 500 -%}{{ section.settings.cover_image | img_url: '500x' }} 500w,{%- endif -%}
+            {%- if section.settings.cover_image.width >= 730 -%}{{ section.settings.cover_image | img_url: '730x' }} 730w,{%- endif -%}
+            {%- if section.settings.cover_image.width >= 1440 -%}{{ section.settings.cover_image | img_url: '1440x' }} 1440w,{%- endif -%}
+            {%- if section.settings.cover_image.width >= 990 -%}{{ section.settings.cover_image | img_url: '990x' }} 990w,{%- endif -%}
+            {%- if section.settings.cover_image.width >= 2080 -%}{{ section.settings.cover_image | img_url: '2080x' }} 2080w{%- endif -%}"
+            src="{{ section.settings.cover_image | img_url: '533x' }}"
+            sizes="(min-width: 990px)990px, (min-width: 750px)calc(100vw - 100px), calc(100vw - 30px)"
+            alt="{{ section.settings.description | escape }}"
+            loading="lazy"
+            width="{{ section.settings.cover_image.width }}"
+            height="{{ section.settings.cover_image.height }}"
+            class="video-section__image"
+          >
+        {%- else -%}
+          {{ 'collection-2' | placeholder_svg_tag: 'placeholder-svg placeholder video-section__image' }}
+        {%- endif -%}
+        <span class="deferred-media__poster-button motion-reduce">
+          {%- render 'icon-play' -%}
+        </span>
+      </button>
+      <template>
+        {%- if section.settings.video_url.type == 'youtube' -%}
+          <iframe src="https://www.youtube.com/embed/{{ section.settings.video_url.id }}?enablejsapi=1" class="js-youtube" allow="autoplay; encrypted-media" allowfullscreen title="{{ section.settings.description | escape }}"></iframe>
+        {%- else -%}
+          <iframe src="https://player.vimeo.com/video/{{ section.settings.video_url.id }}" class="js-vimeo" allow="autoplay; encrypted-media" allowfullscreen title="{{ section.settings.description | escape }}"></iframe>
+        {%- endif -%}
+      </template>
+    </deferred-media>
+  </div>
+</div>
+
+{% schema %}
+{
+  "name": "Video",
+  "tag": "section",
+  "class": "spaced-section spaced-section--full-width",
+  "settings": [
+    {
+      "type": "text",
+      "id": "heading",
+      "default": "Video",
+      "label": "Heading"
+    },
+    {
+      "type": "image_picker",
+      "id": "cover_image",
+      "label": "Cover image"
+    },
+    {
+      "type": "video_url",
+      "id": "video_url",
+      "accept": [
+        "youtube",
+        "vimeo"
+      ],
+      "default": "https:\/\/www.youtube.com\/watch?v=_9VUPq3SxOc",
+      "label": "Link",
+      "placeholder": "Use a YouTube or Vimeo link",
+      "info": "Video will play inside the page"
+    },
+    {
+      "type": "text",
+      "id": "description",
+      "default": "Describe the video for customers using screen readers",
```

Comment:
Wondering if this should become an "info" below the input so merchants can read the full sentence?

Reply:
Good point, so it doesn't go away. And have a default be `Video to describe your products or your brand` kind of thing ?

Reply:
I wonder if there should be a default, if so, it would have to be short otherwise merchants can't read the full sentence. Perhaps: `Describe your video` ?

When we edit alt text of an image this is the info helper text:

> Write a brief description of this image to improve search engine optimization (SEO) and accessibility for visually impaired customers. This change will be saved immediately.

@gretchen-willenberg could we improve the Video alt text info text to align with what we have for image alt text (and ignore the SEO part)?

Draft: `Describe the video for visually impaired customers using screen readers.`

Reply:
I think the reason why users might choose to use screen readers can be left out. Because it's not necessary always true.

Reply:
Sure!
1. Updated the `Video alt text` placeholder copy from repeating `Video alt text` to `Describe the video`
2. Agree with your revision to `Describe the video for customers using screen readers.` so it's `Describe the video for visually impaired customers using screen readers.`

I think I had some fuzzy old memory about not specifying why some one might choose to use a screen reader, but all research I just skimmed includes at least "visually impaired" if not "blind or visually impaired"--so corrected!

Reply:
> I think the reason why users might choose to use screen readers can be left out. Because it's not necessary always true.

@ludoboludo can you elaborate on this please?

Reply:
Added some [research to the Slack thread](https://shopify.slack.com/archives/C03T74ZTD/p1626296045302700) and an updated version of copy: 
`Describe the video to make it accessible for customers using screen readers.`